cddl/examples/comid-domain-mem.diag

```diff
@@ -21,7 +30,15 @@
           }
         ]
       ],
-      [ / domain - int / 1,
+      [   / environment-map / {
```

Comment:
```suggestion
      [  / The root of trust is in the trust domain of both loader modules /
         / environment-map / {
```

Reply:
Again this is the interpretation "I am in x, y, z". Root -> LoadInc module{1,2}

If instead you're meaning to say loader modules 1 and 2 are always under this root of trust, then use text "The LoadInc modules are under the PQR root of trust." LoadInc module{1,2} -> Root